### PR TITLE
Add Breadcrumb to Patient History page

### PR DESCRIPTION
**Pull request by tendomart**
Screenshot from 2024-01-23 09-34-18

Hi @mozzy11  do you mind merging this if all is good

### CODE REVIEW

**Comment by mozzy11** [comment]
Thanks @tendomart 
Can you add the bread crumbs to the rest of the pages where it is missing ??

**Comment by tendomart** [comment]
sure.....I will

File: frontend/src/components/patient/PatientHistory.js [hunk 1]
```diff
@@ -15,6 +16,11 @@ const PatientHistory = () => {
     }
   };
 
+  const [newPatientTab, setNewPatientTab] = useState({
```
Comment (mozzy11) [inline frontend/src/components/patient/PatientHistory.js:19]: @tendomart , i dont see any palce exactly where this state is updated.
Is this still WIP ??

Comment (tendomart) [reply frontend/src/components/patient/PatientHistory.js]: that was removed

**Comment by tendomart** [comment]
The following pages have recieved breadcrumbs SamplePatientEntry, Modify Order, and search incoming Requests.see screenshots below..

will complete others later
SamplePatientEntry
modifyOrder
eorder

**Comment by mozzy11** [comment]
Thanks @tendomart

**Pull request merged**